UMN-AutoPackager/public/Get-GlobalConfig.ps1

```diff
@@ -0,0 +1,30 @@
+function Get-Config {
+    <#
+    .SYNOPSIS
+        Gets all the global configurations and value for each from a JSON file used by the auto packager.
+    .DESCRIPTION
+        This command retrieves from a json file the values of various global configurations used as part of the AutoPackager. It requires a the full path and name of the JSON file.
+    .EXAMPLE
+        Get-Config -Path .\config.json
+        Gets the values of the config.json file
+    .PARAMETER Path
+        The full path and file name of the JSON file to get the config from
+    #>
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$True, HelpMessage = "The full path and file name of the JSON file to get the config from")]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path
+    )
+    begin {
+        Write-Verbose -Message "Starting $($myinvocation.mycommand)"
+    }
+    process {
+        $json = Get-Content $Path -Raw
```

Comment:
I would add in the -Path for the $path variable to make this cleaner.

Reply:
Added switches to any I saw that were missing.